sklearn/cluster/spectral.py

```diff
@@ -222,6 +222,10 @@ def spectral_clustering(affinity, n_clusters=8, n_components=None,
         the 'Multiclass spectral clustering' paper referenced below for
         more details on the discretization approach.
 
+    copy : bool, optional, default: False
+        Parameter to opt whether memory optimization needs to be done in
```

Comment:
default should be True, particularly since that is current behaviour.

Description more like "If False, the values in affinity will be changed, saving memory."